circle.yml

```diff
@@ -76,6 +79,10 @@ test:
     # are much faster, no need to wait for them)
     - while [ ! -e ~/sauce_is_ready ]; do sleep 1; done
 
+    # Start taking screenshots in the background while the unit tests are running
+    - npm install wd && node script/screenshots.js http://localhost:3000/test/visual.html && touch ~/screenshots:
```

Comment:
apparently `npm install` doesn't short-circuit if the package is already installed, so how about `{ test -d node_modules/wd || npm install wd; } && ...` so we can take advantage of the CircleCI cache (I know, I know, it's a meaningless micro-optimization)